app/templates/admin/createSingleEvent.html

```diff
@@ -244,18 +244,22 @@ <h1 id="title">{{page_title}}</h1>
           <div class="col-md-6 offset-md-3">
             <div class="text-center">
               {% if not isPastEvent  %}
-                {% if userHasRSVPed %}
-                  <button type="submit" class="btn btn-danger"  formaction="/rsvpRemove">Remove RSVP</button>
-                {% else %}
-                  <button type="submit" id="rsvpEvent" class="btn btn-primary" formaction="/rsvpForEvent" {{'disabled' if isPastEvent}}>RSVP</button>
-                {% endif %}
-              {% endif %}
-
+                  {% if userHasRSVPed %}
+                    <button type="submit" class="btn btn-danger"  formaction="/rsvpRemove">Remove RSVP</button>
+                  {% else %}
+                     {% if request.path[-6:] != "create"%}
+                     <button type="submit" id="rsvpEvent" class="btn btn-primary" formaction="/rsvpForEvent" {{'disabled' if isPastEvent}}>RSVP</button>
+                     {% endif %}
+                 {% endif %}
+            {% else %}
+                <button type="submit" id="rsvpEvent" class="btn btn-primary disabled" formaction="/rsvpForEvent" >RSVP</button>
+            {% endif %}
```

Comment:
There does not need to be two different rsvp buttons. figure out how to only have the html once. It should only be there if the event is in the future, and if we are not creating a new event

Reply:
I have resolved that and. it should be good now.